script/generate_config_docs.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+set -e
+
+echo "Generating Flyte Configuration Documents"
+CUR_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+ROOT_DIR=${CUR_DIR}/..
+OUTPUT_DIR="${ROOT_DIR}"/rsts/deployment/cluster_config
+GOBIN=${GOPATH:-~/go}/bin
+
+GO111MODULE=on go get github.com/flyteorg/flyteadmin/cmd@latest
```

Comment:
You might want to talk with Yuvraj about how to do this:
Take a look at the "Generate Flyte Manifest" GitHub workflow... This script should run as part of that and should not pull latest versions but rather the same versions that the workflow pulls..

Reply:
done